biodatasets/scai_disease/scai_disease.py

```diff
@@ -0,0 +1,251 @@
+in life sciences.
+"""
+
+_HOMEPAGE = "https://www.scai.fraunhofer.de/en/business-research-areas/bioinformatics/downloads/corpus-for-disease-names-and-adverse-effects.html"
+
+_LICENSE = ""
+
+_URLS = {
+    _DATASETNAME: "https://www.scai.fraunhofer.de/content/dam/scai/de/downloads/bioinformatik/Disease-ae-corpus.iob",
+}
+
+_SUPPORTED_TASKS = [Tasks.NAMED_ENTITY_RECOGNITION]
+
+_SOURCE_VERSION = "1.0.0"
+
+_BIGBIO_VERSION = "1.0.0"
+
+
+class ScaiDiseaseDataset(datasets.GeneratorBasedBuilder):
+    """SCAI Disese is a dataset annotated in 2010 with mentions of diseases and
+    adverse effects."""
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    BIGBIO_VERSION = datasets.Version(_BIGBIO_VERSION)
+
+    # You will be able to load the "source" or "bigbio" configurations with
+    # ds_source = datasets.load_dataset('my_dataset', name='source')
+    # ds_bigbio = datasets.load_dataset('my_dataset', name='bigbio')
+
+    BUILDER_CONFIGS = [
+        BigBioConfig(
+            name="scai_disease_source",
+            version=SOURCE_VERSION,
+            description="SCAI Disease source schema",
+            schema="source",
+            subset_id="scai_disease",
+        ),
+        BigBioConfig(
+            name="scai_disease_bigbio_kb",
+            version=BIGBIO_VERSION,
+            description="SCAI Disease BigBio schema",
+            schema="bigbio_kb",
+            subset_id="scai_disease",
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = "scai_disease_source"
+
+    def _info(self) -> datasets.DatasetInfo:
+        if self.config.schema == "source":
+            features = datasets.Features(
+                {
+                    "document_id": datasets.Value("string"),
+                    "text": datasets.Value("string"),
+                    "tokens": [
+                        {
+                            "offsets": [datasets.Value("int64")],
+                            "text": datasets.Value("string"),
+                            "tag": datasets.Value("string"),
+                        }
+                    ],
+                    "entities": [
+                        {
+                            "offsets": [datasets.Value("int64")],
+                            "text": datasets.Value("string"),
+                            "type": datasets.Value("string"),
+                        }
+                    ],
+                }
+            )
+
+        elif self.config.schema == "bigbio_kb":
+            features = schemas.kb_features
+        else:
+            raise ValueError("Unrecognized schema: %s" % self.config.schema)
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager) -> List[datasets.SplitGenerator]:
+        """Returns SplitGenerators."""
+        urls = _URLS[_DATASETNAME]
```

Comment:
small thing, but for our future selves ... could we make `urls` singular? when you do `_URLS[_DATASETNAME]` you just get back a single string that is one url right?